sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala

```diff
@@ -403,17 +400,18 @@ object CreateDataSourceTableUtils extends Logging {
       assert(partitionColumns.isEmpty)
       assert(relation.partitionSchema.isEmpty)
 
+      var storage = CatalogStorageFormat(
+        locationUri = None,
```

Comment:
Any reason why this `locationUri` is set to `None`? It sounds like the original value is `Some(relation.location.paths.map(_.toUri.toString).head`